### PR TITLE
BZ2068397 Updating machine_management/modules/machineset-yaml-aws.adoc

### PR DESCRIPTION
**Pull request by agantony**
Version(s): 
4.8+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2068397

Link to docs preview:
https://55373--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-aws.html#machineset-yaml-aws_creating-machineset-aws

QE review:
- [x] QE has approved this change.

Additional information:
Merge into `main` and cherry-pick into:
- `enterprise-4.13`
- `enterprise-4.12`
- `enterprise-4.11`
- `enterprise-4.10`
- `enterprise-4.9`
- `enterprise-4.8`

### CODE REVIEW

**Comment by agantony** [comment]
QE review needed: @minmli@redhat.com

**Comment by lyman9966** [comment]
Hi, @agantony 
I saw you modified all the "machine set" to "compute machine set", do we need emphasize "compute"? The operation is unified to the crd MachineSet. And what's the opposite side of "compute"? we don't mention this

File: modules/machineset-yaml-aws.adoc [hunk 1]
```diff
@@ -144,8 +144,13 @@ $ oc -n openshift-machine-api \
 ====
 Custom tags can also be specified during installation in the `install-config.yml` file. If the `install-config.yml` file and the machine set include a tag with the same `name` data, the value for the tag from the machine set takes priority over the value for the tag in the `install-config.yml` file.
 ====
+
+<6> Specify the zone, for example, `us-east-1a`.
+<7> Specify the region, for example, `us-east-1a`.
```
Comment (lyman9966) [inline modules/machineset-yaml-aws.adoc:149]: <7> the region example should be: us-east-1 (not us-east-1a)

**Comment by agantony** [comment]
> Hi, @agantony I saw you modified all the "machine set" to "compute machine set", do we need emphasize "compute"? The operation is unified to the crd MachineSet. And what's the opposite side of "compute"? we don't mention this

I have only added callouts in the YAML file. Please take a look at the 

> Hi, @agantony I saw you modified all the "machine set" to "compute machine set", do we need emphasize "compute"? The operation is unified to the crd MachineSet. And what's the opposite side of "compute"? we don't mention this

I have only added callouts in the YAML file. 
Please take a look at the commit: https://github.com/openshift/openshift-docs/pull/55373/commits/5ab536e29f15759f60303f0163bb73c1e9b5b845

File: modules/machineset-yaml-aws.adoc (hunk 1 above)
Comment (agantony) [reply modules/machineset-yaml-aws.adoc:149]: ack, updated the doc.

**Comment by lyman9966** [comment]
> > Hi, @agantony I saw you modified all the "machine set" to "compute machine set", do we need emphasize "compute"? The operation is unified to the crd MachineSet. And what's the opposite side of "compute"? we don't mention this
> 
> I have only added callouts in the YAML file. Please take a look at the commit: [5ab536e](https://github.com/openshift/openshift-docs/commit/5ab536e29f15759f60303f0163bb73c1e9b5b845)

Sorry, I compared the chapter title with OCP 4.10, indeed 4.12 has updated the title.

**Comment by Amrita42** [comment]
lgtm

**Comment by agantony** [comment]
> @agantony I am getting merge errors with older versions of this file. I tried 4.8 and 4.11. I visually scanned the file in 4.8 - 4.11 and there are differences in those files from this one. For example, some already have an infra taints (in a different location) and custom tags seem to start in 4.12. I think someone needs to make the appropriate changes one version at a time, or carefully cherrypick. Is that something you can do? I am not comfortable making the changes myself.

Hi @mburke5678 I will take a look at this and see if I can fix it myself :+1:

**Comment by jeana-redhat** [comment]
@agantony thanks so much for handling this update - it has been on my enhancements backlog for a little bit. I am sorry you got to it before I had a chance to make the `ifdef` conditions less annoying to work with (also in my backlog) :pray:

**Comment by agantony** [comment]
> @agantony thanks so much for handling this update - it has been on my enhancements backlog for a little bit. I am sorry you got to it before I had a chance to make the `ifdef` conditions less annoying to work with (also in my backlog) pray

No problem @jeana-redhat :-)

**Pull request merged**